pages/linux/scan-build.md

```diff
@@ -0,0 +1,35 @@
+# scan-build
+
+> scan-build is a command line utility that enables a user to run the
+> static analyzer over their codebase as part of performing a regular
+> build (from the command line).
+> More information: <https://clang-analyzer.llvm.org/scan-build.html>.
+
+Basic usage of scan-build is designed to be simple: just place the word
+`scan-build` in front of your build command:
```

Comment:
We use this format for example descriptions
```suggestion
- Build and analyze the project in the current directory:
```

Reply:
Fixed :heavy_check_mark: